logging_utilities/filters/django_request.py

```diff
@@ -42,32 +46,36 @@ def __init__(self, include_keys=None, exclude_keys=None):
                 All request attributes that match any of the dotted keys of the list will not be
                 added to the jsonify of the record.request. NOTE this has precedence to include_keys
                 which means that if a key is in both list, then it is not added.
+            attr_name: str
+                The name of the attribute that stores the HttpRequest object. The default is
+                'request', as Django already sometimes logs a HttpRequest under this attribute.
+                Beware however that Django sometimes also logs an object of type 'socket' under this
+                attribute, which could cause problems if you don't filter it away correctly.
```

Comment:
If using `http_request` no issue here. Also a comment about Django logging the HTTP Request under `request` is worth mentioning.